gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -324,7 +324,7 @@
             </div>
           </div>
 
-          <div class="row" ng-if="currentFolder.syncXattrs">
+          <div class="row" ng-if="currentFolder.syncXattrs || currentFolder.sendXattrs">
```

Comment:
```suggestion
          <div class="row" ng-if="currentFolder.sendXattrs">
```

Haven't tested but wouldn't just `currentFolder.sendXattrs` suffice? Because "sync" always incorporates "send".

Reply:
Looks like it would, though I'm not sure it's worth depending on it. Checking both like that is a GUI clarification, in the actual config it's fully possible to have just sync but not send checked.